clevertap-core/src/main/java/com/clevertap/android/sdk/pushnotification/CoreNotificationRenderer.java

```diff
@@ -178,6 +178,21 @@ private Builder finalizeBuilder(Builder nb, Bundle extras, Context context,
             nb.setColorized(true);
         }// uncommon
 
+        String dismissValue = extras.getString(Constants.WZRK_DISMISS);
+        if (dismissValue != null) {
+            try {
+                long dismissAfter = Long.parseLong(dismissValue) * 1000L;
+                nb.setTimeoutAfter(dismissAfter);
+            } catch (NumberFormatException e) {
+                config.getLogger().verbose(config.getAccountId(), "Failed to parse timeout dismiss value", e);
+            }
+        }
+
+        boolean isSticky = "true".equalsIgnoreCase(extras.getString(Constants.WZRK_STICKY));
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.UPSIDE_DOWN_CAKE) {
+            nb.setOngoing(isSticky);
```

Comment:
@Anush-Shand Not sure if this will work, https://developer.android.com/about/versions/14/behavior-changes-all#non-dismissable-notifications